src/System.Private.CoreLib/src/System/Exception.CoreCLR.cs

```diff
@@ -627,24 +449,12 @@ internal void RestoreDispatchState(in DispatchState dispatchState)
         // @MANAGED: HResult is used from within the EE!  Rename with care - check VM directory
         private int _HResult;       // HResult
 
-        public int HResult
-        {
-            get
-            {
-                return _HResult;
-            }
-            set
-            {
-                _HResult = value;
-            }
-        }
-
         private string _source;         // Mainly used by VB. 
         // WARNING: Don't delete/rename _xptrs and _xcode - used by functions
         // on Marshal class.  Native functions are in COMUtilNative.cpp & AppDomain
         private IntPtr _xptrs;             // Internal EE stuff 
 #pragma warning disable 414  // Field is not used from managed.
-        private int _xcode;             // Internal EE stuff 
+        private int _xcode = _COMPlusExceptionCode;             // Internal EE stuff 
```

Comment:
Just for reference:

This is a small behavior difference: This field was not set before this change for deserialized exceptions. I do not see a problem with it.